karma-modules-ngAnimate.conf.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var angularFiles = require('./angularFiles');
+var sharedConfig = require('./karma-shared.conf');
+
+module.exports = function(config) {
+  sharedConfig(config, {testName: 'AngularJS: isolated module tests', logFile: 'karma-modules-isolated.log'});
```

Comment:
the name and the log files (not that we access them ...) should be specific to ngAnimate, ngMock / other. That makes it easier to distinguish them on Saucelabs.

Reply:
Done